src/test/java/DemoScript.java

```diff
@@ -0,0 +1,128 @@
+import com.smartbear.visualtest.VisualTest;
```

Comment:
Create a module inside the `visualtest-java-examples` folder called `TestNG-Parallel` and move the code inside that module. so in future, we can create multiple modules for different frameworks like JUnit, Cucumber etc.